### ISSUE TITLE
Internal Console is not displaying stdout output on node 8.1

### ISSUE THREAD

**Issue by rluncasu** [Microsoft/vscode#28872]
Version 1.14.0-insider (1.14.0-insider)

my launch.json
```json
        {
            "type": "node",
            "request": "launch",
            "protocol": "inspector",
            "name": "Launch Server",
            "program": "${workspaceRoot}/server.js",
            "env": {
                "DEBUG": "*",
                "NODE_ENV": "development"
            }
        }
```

Debug console output:
```sh
node --inspect=35293 --debug-brk server.js 
Debugger listening on ws://127.0.0.1:35293/695167d4-ae99-401a-b30f-92228049ed94
Debugger attached.
```
## Couple of issues here:
- although the option seems to be --debug-brk , it doesn't break on first line
- there's no output whatsoever from stdout

## Workaround:

Use the integrated terminal as a console by adding:
```json
"console": "integratedTerminal",
```
to the launch configuration.

## PS:
```sh
[DEP0062] DeprecationWarning: `node --inspect --debug-brk` is deprecated. Please use `node --inspect-brk` instead
```
(from the integrated terminal output)

**Comment by weinand** [comment]
@rluncasu explanation for your first issue:
We always use "--debug-brk" in order to register breakpoints on startup without node.js already executing the program at the same time (and potentially missing some breakpoints).
But we automatically "continue" over the 'break on entry' If there is no `"stopOnEntry": "true"` in the launch config. So, that you don't see a "stop on entry" is correct and works as designed.

**Comment by roblourens** [comment]
Can you give a code snippet that doesn't show output? Are you using `console.log` or `process.stdout.write`? If the latter, this would be https://github.com/Microsoft/vscode/issues/19750

**Comment by rluncasu** [comment]
Thanks @weinand , that makes sense.
Thanks @roblourens , I'm using a logger which I investigated today and indeed it boils down to process.stdout.write . switched to console.log and it's fine now.

**Comment by saeedtabrizi** [comment]
Hi @roblourens , i have same issue with same configuration in **vscode 1.14.2** .
when i use vscode **internalConsole:true** option and using the following code :
```
process.stdout.write("Not printed");
console.log("printed");
```
I can see only "printed" message in debug console . 
When this bug can be fix ? i see an open issue #19750 related to it , but i don't know when this problem can be fixed in vscode .

Thanks
